demos/src/simple-header.mustache

```diff
@@ -0,0 +1,60 @@
+<header class="o-header o-header--simple" data-o-component="o-header" data-o-header--no-js>
+    <div class="o-header__row o-header__top">
+        <div class="o-header__container">
+            <div class="o-header__top-wrapper">
+                <div class="o-header__top-column o-header__top-column--left">
```

Comment:
Worth adding the mustache ifs for menu, search and stuff as they're set in the data file